app/views/collections/index.html.erb

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
```

Comment:
I was struggling to get the layout to play nicely for this standalone view without adding in the extra html. I know it's not idea, but I needed to get this read for testing.